install/helm/open-match/values.yaml

```diff
@@ -106,11 +106,13 @@ configs:
 redis:
   fullnameOverride: om-redis
   redisPort: 6379
-  usePassword: true
-  usePasswordFile: true
+  usePassword: false
+  usePasswordFile: false
   secretMountPath: /opt/bitnami/redis/secrets
   ignoreLists:
     ttl: 60000ms
+  configmap:
+    maxclients 100000
```

Comment:
Not in this PR but our best practices / scaling overview doc should list what the tradeoffs here are - for example - based on scenarios, how should these change?

Reply:
Sure, will add this as a note to the issue that tracks the production guide.